.github/workflows/all-pr-tests.yml

```diff
@@ -0,0 +1,43 @@
+name: "All Pull Request Tests"
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  # We use a single job to ensure that all steps run in the same environment and
+  # reduce the number of minutes used.
+  pr-tests:
+    # Don't run on draft PRs
+    if: github.event.pull_request.draft == false
+    # Timeout after 10 minutes
+    timeout-minutes: 10
+    # Define a matrix of PHP/WordPress versions to test against
```

Comment:
Should include the `concurrency` block with the PHP and WordPress versions included in what defines the group. See https://github.com/alleyinteractive/create-wordpress-project/blob/develop/ci-templates/.github/workflows/all-pr-tests.yml#L18